verification/policy.go

```diff
@@ -125,9 +125,11 @@ func validateTrustedIdentities(statement TrustPolicy) error {
 func validateTrustStore(statement TrustPolicy) error {
 	supportedTrustStorePrefixes := []string{"ca"}
 
-	i := strings.Index(statement.TrustStore, ":")
-	if i < 0 || !isPresent(statement.TrustStore[:i], supportedTrustStorePrefixes) {
-		return fmt.Errorf("trust policy statement %q uses an unsupported trust store type %q in trust store value %q", statement.Name, statement.TrustStore[:i], statement.TrustStore)
+	for _, trustStore := range statement.TrustStores {
+		i := strings.Index(trustStore, ":")
+		if i < 0 || !isPresent(trustStore[:i], supportedTrustStorePrefixes) {
+			return fmt.Errorf("trust policy statement %q uses an unsupported trust store type %q in trust store value %q", statement.Name, trustStore[:i], trustStore)
+		}
```

Comment:
It seems this block of code does not support case like `ca:acme-rockets,tsa:acme-tsa`.

Reference: https://github.com/notaryproject/notaryproject/pull/164/files#r914176243
/cc @gokarnm

Reply:
Yes, I'm aware of this. Not parsing TSA trust stores yet as TSA verification is out of scope for RC1. Will create an issue for tracking

Reply:
https://github.com/notaryproject/notation-go/issues/74